pilot/cmd/pilot-agent/status/server.go

```diff
@@ -83,24 +94,26 @@ type Prober struct {
 
 // Config for the status server.
 type Config struct {
-	LocalHostAddr string
+	PodIp string
```

Comment:
This can be vm

Reply:
The app prober stuff only runs on k8s though?